src/style.css

```
@@ -0,0 +1,80 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars--0 * {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```

```suggestion
  background-repeat: no-repeat;
  background-position: center;
```
Дуже багато дублювання. Ці два властивості потрібно додати до .stars__star, адже вони стосуються всіх зірок, і більше ніде не писати